shared-bindings/microcontroller/__init__.c

```diff
@@ -58,17 +58,19 @@
 //|     :maxdepth: 3
 //|
 //|     Pin
+//|     Processor
 //|
 
 //| .. attribute:: cpu
 //|
-//|   CPU information and control, such as temperature and clock frequency.
+//|   CPU information and control, such as `cpu.temperature` and `cpu.frequency`
+//|   (clock frequency).
 //|   This object is the sole instance of `microcontroller.Processor`.
 //|
 
 //| .. method:: delay_us(delay)
 //|
-//|   Dedicated delay method used for very short delays. DO NOT do long delays
+//|   Dedicated delay method used for very short delays. **Do not** do long delays
```

Comment:
If you have a ballpark on what is long or short, that may help end users.

Reply:
That's a good question. Probably this should explain further that it basically shuts off everything while the delay is happening, and let the user decide whether that's OK or not. It depends on whether delaying some I/O is going to be a problem or not, such a NeoPixel timing or USB input.